### ISSUE TITLE
Catching & dragging moving scrollbar after swipe strange behavior

### ISSUE THREAD

**Issue by mz3r0** [kivy/kivy#8930]
**Software Versions**
* Python: 3.12.0
* OS: Windows 10 Pro Version 22H2 Build 19045.5131
* Kivy: 2.3.0
* Kivy installation method: pip

**Describe the bug**
Swiping and catching the moving scrollbar causes strange behavior.

It looks like the KineticScroll effect isn't reset when scrolling the handle if grabbed while moving.

Conflict likely arises due to more than one function simultaneously attempting to control the scrollbar movement.

**Expected behavior**
I expect the velocity to be reset when grabbing the scrollbar.

**To Reproduce**
A short, *runnable* example that reproduces the issue with *latest kivy master*.

**Code and Logs and screenshots**

Check this gist: https://gist.github.com/mz3r0/4aee7a6623016c90a7c65e9cb05daf47

**Comment by kuzeyron** [comment]
On my end this works as it should with `v2.3.1.dev0`.
Try `v2.3.1.dev0` and post the results.

**Comment by mz3r0** [comment]
I tried it and has the same behavior. See mp4 at https://file.io/q4kWiBasGZbp

**Comment by kuzeyron** [comment]
> I tried it and has the same behavior. See mp4 at https://file.io/q4kWiBasGZbp

"It's deleted".

**Comment by mz3r0** [comment]
Apologies, https://filebin.net/6588q4k4buapb4y4

**Comment by kuzeyron** [comment]
Is it the same with this usage?
```YML
ScrollView:
        effect_cls: 'ScrollEffect'
```

**Comment by mz3r0** [comment]
It already does that. In the gist, Line 20 is `self.effect_cls = ScrollEffect`. Doing it kv-style doesn't change anything though

**Comment by kuzeyron** [comment]
> It already does that. In the gist, Line 20 is `self.effect_cls = ScrollEffect`. Doing it kv-style doesn't change anything though

Right. I was just quickly reading through that gist paste and further down in the code I didn't see anything about that.
